eng/validation/update-channel.ps1

```diff
@@ -2,7 +2,7 @@ Param(
   [string] $maestroEndpoint,
   [string] $barToken,
   [string] $apiVersion = "2018-07-16",
-  [string] $targetChannelName = ".NET Eng - Latest",
+  [string] $targetChannelName = ".NET 6 Eng",
```

Comment:
I'm confused, I thought this was already fine in the branch? https://github.com/dotnet/arcade-validation/blob/release/6.0/eng/validation/update-channel.ps1#L5

Reply:
Oh the PR was closed. Nevermind.